packages/webview_flutter/webview_flutter_android/lib/webview_android_widget.dart

```diff
@@ -134,6 +143,11 @@ class WebViewAndroidPlatformController extends WebViewPlatformController {
   /// This should only be changed for testing purposes.
   final WebViewProxy webViewProxy;
 
+  /// Manages access to Flutter assets that are part of the Android App bundle.
+  ///
+  /// This should only be changed for testing purposes.
```

Comment:
Sounds like it should be annotated `@visibleForTesting` then

Reply:
My reasoning was that since the member is marked `final` it can only be set through the constructor and the constructor parameter is already set to `@visibleForTesting` it wouldn't be necessary to mark this as visible for testing (to be consistent with the `webViewProxy` field).